sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/AADAuthenticationProperties.java

```diff
@@ -20,14 +20,17 @@
 import java.util.Optional;
 import java.util.concurrent.TimeUnit;
 
+import static com.azure.spring.autoconfigure.aad.AADAuthenticationProperties.AAD_PREFIX;
+
 /**
  * Configuration properties for Azure Active Directory Authentication.
  */
 @Validated
-@ConfigurationProperties("azure.activedirectory")
+@ConfigurationProperties(AAD_PREFIX)
```

Comment:
Can we keep the same style? All use `AAD_PREFIX`, or not use it at all.

Ofcourse, we can do it after release.

Reply:
Which one do you prefer?

Reply:
I prefer not use it at all. If we want to get some property, we can search by `CTRL + SHIFT + F` to get them all.

Reply:
okay